packages/nc-gui-v2/plugins/state.ts

```diff
@@ -25,6 +26,12 @@ export default defineNuxtPlugin((nuxtApp) => {
   /** set i18n locale to stored language */
   nuxtApp.vueApp.i18n.locale.value = state.lang.value
 
+  try {
+    state.appInfo = await $api.utils.appInfo()
```

Comment:
should probably be `state.appInfo.value` 
Yo could also just destructure `state`

```ts
const { appInfo } = useGlobal()

...

appInfo.value = ...
```